src/scale_impls/mod.rs

```diff
@@ -16,8 +16,6 @@
 mod decode;
 mod encode;
 
-pub use decode::{decode_value_as_type, DecodeError, DecodeValueVisitor};
-pub use encode::EncodeError;
+pub use decode::{decode_any_value_as_type, decode_value_as_type, DecodeError};
 
-/// A type ID which can be resolved into a type given a [`::scale_info::PortableRegistry`].
-pub type TypeId = u32;
+// pub type TypeId = u32;
```

Comment:
remove or why not keep it?

Reply:
I did not want to keep it because the amount of places where the type alias is used is now quite limited, because most of the code uses `R::TypeId`. Having another item named `TypeId` was a bit confusing.

Reply:
cool